app-web/plugins/gatsby-source-github-all/utils/plugins.js

```diff
@@ -95,8 +96,10 @@ const markdownPagePathPlugin = (extension, file) => {
   } else {
     // no resource path, this file is destined to be turned into a page,
     // the page page is composed of the source name, the title of the file plus an id
-    file.metadata.resourcePath = `/${file.metadata.source}/${file.metadata
-      .name}_${shortid.generate()}`;
+    file.metadata.resourcePath = createPathWithDigest(
```

Comment:
You could probably just pass in `file.metadata.source` and `file.metadata.name` as the two params then `+` them as needed in the function.

Reply:
updated the function to take a spread of strings (basePath, digestable1, digestable2 ...etc). The spread of strings are joined.